src/ImageSharp/Processing/Processors/Filters/BlackWhiteProcessor.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using SixLabors.ImageSharp.PixelFormats;
+
+namespace SixLabors.ImageSharp.Processing.Processors
+{
+    /// <summary>
+    /// Applies a black and white filter matrix to the image
+    /// </summary>
+    /// <typeparam name="TPixel">The pixel format.</typeparam>
+    internal class  BlackWhiteProcessor<TPixel> : FilterProcessor<TPixel>
+          where TPixel : struct, IPixel<TPixel>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BlackWhiteProcessor{TPixel}"/> class.
+        /// </summary>
+        public BlackWhiteProcessor()
```

Comment:
I wonder if we still need these subclasses?

Reply:
Some yes, some no. I'm happy to drop the ones we don't leaving only the ones that require extra parameters.